src/common/cockpitpipe.c

```diff
@@ -372,6 +374,24 @@ dispatch_input (gint fd,
   return TRUE;
 }
 
+static void
+forward_error (CockpitPipe *self)
+{
+  CockpitPipePrivate *priv = cockpit_pipe_get_instance_private (self);
+
+  guint8 *data = priv->err_buffer->data;
+  guint len = priv->err_buffer->len;
+  while (len > 0)
+    {
+      ssize_t n = TEMP_FAILURE_RETRY (write (2, data, len));
+      if (n <= 0)
+        break;
```

Comment:
I guess at the very least, if we get an unhandled error then we should probably stop attempting to continue forwarding...

Reply:
We might even kill the whole bridge, no?  Getting an error here means that stderr is hosed, and we shouldn't probably not continue at all with the session, no?  What does `g_log` do if there is an error writing to stderr?  Actually, we could just use `g_log` to handle all of this for us, no?